test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/017_Change_audio_source_from_any_type_except_MOBILE_APP_keepContext_true.lua

```diff
@@ -34,7 +34,10 @@ local audioSources = {
   "USB2",
   "LINE_IN",
   "IPOD",
-  "RADIO_TUNER"
+  "AM",
+  "FM",
+  "XM",
+  "DAB"
 }
```

Comment:
Use `common.audioSources`

Reply:
@jacobkeeler , common.audioSources contains "MOBILE_APP" value, and this script tests all types except "MOBILE_APP", so here array is defined localy without "MOBILE_APP".